webpack.config.js

```diff
@@ -3,7 +3,9 @@ var path = require('path'),
     merge = require('webpack-merge'),
     HtmlWebpackPlugin = require('html-webpack-plugin'),
     CleanWebpackPlugin = require('clean-webpack-plugin');
-    
+
+require('es6-promise');
```

Comment:
I was having issues with webpack and it kept throwing issue around promises. I would have to uncomment it out to tell you the exact issue.